### ISSUE TITLE
Add reflection support for wrappers

### ISSUE THREAD

**Issue by stephenberry** [stephenberry/glaze#549]
Develop a generic way to handle reflection for wrappers like `glz::quoted`, `glz::raw_string`, etc.

**Comment by stephenberry** [comment]
I'm torn about adding reflection support to wrappers, because wrappers work on lambda functions and member functions, making the code more complex. And, I don't want to slow down compile time for normal usage, needing to check if types are wrappers. It keeps the code simpler (and compilation likely faster) by not supporting reflection with wrappers. Users can make their own local macros if they really have issues with the extra typing for wrappers.

In conclusion, I don't think this is a good idea and I'll close this issue.
